src/features/make_canon_dataset.py

```diff
@@ -225,36 +171,19 @@ def group_by_date(cr_con, aggregated):
 
     print("Data directory: " + DATA_FP)
 
-    aggregated, adjacent, cr_con = aggregate_roads(
```

Comment:
Confused: Does this remove the making of adjacency information?

Reply:
The adjacency info generated here is only referenced in the python2/crash_predict_benchmark notebook. The adjacency info that I added more recently is generated earlier in the pipeline and stored as features on the maps, so I thought this was worth removing